LiveSplit.SuperMeatBoy.asl

```diff
@@ -23,20 +23,16 @@ startup
 {
 	settings.Add("menuReset", false, "Reset on main menu");
 	settings.Add("individualLevels", false, "Split after each level");
-	
 	settings.Add("bossSplit", false, "Split when entering selected bosses");
-	for (int world = 1; world <= 6; world++)
-	{
-		string name = String.Format("boss{0}Split", world);
-		string description = String.Format("Split before boss {0}", world);
-		settings.Add(name, false, description, "bossSplit");
-	}
-	
+		for (int world = 1; world <= 6; world++)
+		{
+			string name = String.Format("boss{0}Split", world);
+			string description = String.Format("Split before boss {0}", world);
+			settings.Add(name, false, description, "bossSplit");
+		}
 	settings.Add("deathDisp", false, "Death count display");
-	
 	settings.Add("ilDisp", false, "Last IL Time display");
-	settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
-	
+		settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
```

Comment:
I'm a bit confused by these formatting changes?

Reply:
I arranged the settings in a hierarchy, so any settings/code that are a subset of their parent setting are indented

Reply:
I put the address list in ascending order with the addresses all lined up, which is a format I've found to be very readable and easy to work with

Reply:
The address list is a nice change, but additional indenting I think can be a bit misleading, making it look like a code block. I'll merge this right now, though.